opentreemap/opentreemap/__init__.py

```diff
@@ -0,0 +1,5 @@
+from __future__ import absolute_import, unicode_literals
```

Comment:
Yes. This is from the Celery guide on Django integration: http://docs.celeryproject.org/en/latest/django/first-steps-with-django.html